server/src/Infrastructure/Persistence/ApplicationDbContext.cs

```diff
@@ -25,4 +25,11 @@ protected override void OnModelCreating(ModelBuilder modelBuilder)
 
         base.OnModelCreating(modelBuilder);
     }
+
+    protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
```

Comment:
It should be moved to dependency injection like this `services.AddDbContext<ApplicationDbContext>(options =>
        options.UseSqlServer(configuration.GetConnectionString("DefaultConnection"))
        .AddInterceptors(new AuditableEntityInterceptor()));`